tutorials/katacoda/1-serving-intro/step3.md

```diff
@@ -0,0 +1,28 @@
+## Blue/Green deploy
+The Knative `service` resource creates additional resources "route, configuration and revision" to manage the lifecycle of the application.
```

Comment:
This section might be confusing for new users.

- Title mentions "Blue/Green" but there is not rollout from current version (blue) to the new version (green)
- The traffic is left at 50/50 traffic split, the new version never goes to 100%
- A tag is used in the example but the tag is never use, the tag is for the purpose of unique URL, but it was never use
- There is a command that feels like brack magic `kn revision list -o 'jsonpath={.items[0].metadata.name}`

Here is a [tutorial](https://github.com/csantanapr/knative-tekton#33-knative-service-traffic-splitting) with a flow that is more appropriate. 

We could use a revision name called it revision `blue`, and a revision `green` green being latest.
You can show that green version gets 50% traffic and blue 50%, and then show blue takes 0% and green 100%

Reply:
This was super helpful. I have incorporated this. Thanks!